app/views/jobseekers/job_applications/review/_equal_opportunities.html.slim

```diff
@@ -9,12 +9,10 @@
       - summary_list.row do |row|
         - row.key text: t("helpers.legend.jobseekers_job_application_equal_opportunities_form.disability")
         - row.value text: job_application.disability.humanize
-        - row.html_attributes { { id: "disability" } }
```

Comment:
I think removing this will mean that, when there is an error related to the `disability` field in the error summary / action required box, the link will not go anywhere. So can you add the divs in this partial as you did in the others? The same applies to a couple of other sections.

Reply:
Ah right, I think I understand what you mean. I had been looking at the **"There is a problem"** error summary for the job application review page and saw that it didn't seem to rely on the row ids to link to the errors (those row.html_attributes weren't even successfully adding any ids to the rows) so thought we didn't need them.

I've just seen [your PR](https://github.com/DFE-Digital/teaching-vacancies/pull/3717) and realised we can get an **"Action required"** summary here too, just like the job listing review page which does indeed need ids!